protos/generate.sh

```diff
@@ -68,14 +77,24 @@ for PKG in $PKGS; do
     echo "]" >> "$PKG_INIT"
   done
 
-echo "Done"
+echo "Finish generating protobuf python code"
+
+if ! [ -x "$(command -v pbjs)" ]; then
```

Comment:
let's move all the deps installation to a separate dockerfile so that when running generate-docker locally it can reuse the image and build a lot faster